packages/s3-deploy/lib/S3Ops.mjs

```diff
@@ -94,20 +100,29 @@ class S3Ops {
     const Metadata = this.fileToMetadata(file, hash);
     const ContentEncoding = this.fileToContentEncoding(file);
     const fileName = this.toFile(dir, file);
+    const isNoCacheKey = Key.match(noCachePattern);
+    const CacheControl = isNoCacheKey ? "no-cache" : undefined;
+    if( isNoCacheKey ) {
+      console.log("no-cache set on", Key);
+    }
     const Body = fs.createReadStream(fileName);
     const command = new PutObjectCommand({
       Body,
       Bucket: this.group.Bucket,
       ContentType,
       ContentEncoding,
       Key,
+      CacheControl,
       Metadata,
       ContentSize,
     });
     console.log("uploading", file, ContentType, ContentEncoding, Key, ContentSize, Metadata, this.group.Bucket);
+    if( this.options.dryRun ) {
```

Comment:
@dsmith1547 You might find the --dry-run option useful when testing automated upload.

Reply:
Yes, thanks!